apps/fiona/fiona.cpp

```diff
@@ -79,11 +79,12 @@
 
     #if defined(STDLIB_GNU)
         #include <parallel/algorithm>
-        #define SORT ::__gnu_parallel::sort
+        #define PARALLELTAG Parallel()
     #else
-        #define SORT ::std::sort
+        #define PARALLELTAG Serial()
```

Comment:
This you don't need anymore with the seqan implementation. There is a parallel overload for non gnu_parallel version simply calling the sequential sort.
Just specifying the Parallel tag in the function call is sufficient. It will cover the same behaviour as before, but with less boilerplate.